response_operations_ui/forms.py

```diff
@@ -105,10 +105,10 @@ class UpdateEventDateForm(FlaskForm):
 
     HOURS = [(hour, hour) for hour in ['00', '01', '02', '03', '04', '05', '06', '07', '08', '09', '10', '11',
                                        '12', '13', '14', '15', '16', '17', '18', '19', '20', '21', '22', '23']]
-    hour = SelectField('hours', choices=HOURS)
+    hour = SelectField('hours', choices=HOURS, default='07')
 
     MINUTES = [('00', '00'), ('15', '15'), ('30', '30'), ('45', '45')]
-    minute = SelectField('minutes', choices=MINUTES)
+    minute = SelectField('minutes', choices=MINUTES, default='00')
     submit = SubmitField('Save')
 
     def validate_day(form, field):
```

Comment:
I know its not part of your PR but its a quick fix and could be quickly done now since your editing the file now? Whenever validating certain attributes for the CE's we're passing in the unused param form often would be worth removing i.e. validate_period etc.
https://github.com/ONSdigital/response-operations-ui/pull/190/files#diff-8e743c208190a309d86fe86016048666R127

Reply:
This happens in a few places, probably worth raising a separate PR

Reply:
I think its a minor fix and could be be easily done now slowly rather then a separate card put at the bottom of a column that might be never gotten around to?

Reply:
I think this is a Flask(-wtf) thing rather than a mistakenly added parameter, @MIslam92.

Reply:
@jcox-dev Yeah so should it not just have the default `_` passed in wherever this is the case (unused params) based off your previous comment?

Reply:
If you must. But this follows the expected form of overriding an existing method's definition.

Reply:
@MIslam92 If you must. But this follows the expected form of overriding an existing method's definition.